saleor/graphql/shipping/mutations/shippings.py

```diff
@@ -493,9 +494,7 @@ class Meta:
 
     @classmethod
     def perform_mutation(cls, _root, info, **data):
-        shipping_method = cls.get_node_or_error(
-            info, data.get("id"), only_type=ShippingMethod
-        )
+        shipping_method = get_shipping_model_by_object_id(data.get("id"))
```

Comment:
as above ☝️  but this is better example.
```
def get_shipping_model_by_object_id(
    object_id: Optional[str],
) -> Optional[ShippingMethod]:
    if object_id:
        _, object_pk = from_global_id_or_error(object_id)
        return ShippingMethod.objects.filter(pk=object_pk).first()
    return None
...
497: shipping_method is None
498: shipping_method_id = shipping_method.id <-- shipping_method is None so it will raise an unexpected error. 
```

Reply:
This function throws an exception by default now.